src/reducers/flow.reducer.ts

```diff
@@ -34,6 +36,13 @@ function combinedReducer(state = defaultState, action: any) {
 	);
 }
 
+function roundByStep(type: string, number: number, step: number) {
+	if (type === 'down') {
```

Comment:
up & down should be constant rather than plain strings and define as a type union